### ISSUE TITLE
'Solution' same as another puzzle

### ISSUE THREAD

**Issue by Accudio** [NUKnightLab/sql-mysteries#7]
<details>
<summary>Spoiler: Don't read if you haven't completed!</summary>
Although I do appreciate the reference as someone who has played CLMystery previously as soon as I saw the name Jeremy Bowers in a list of a few 'suspects' I was fairly sure that was it. 

Not as big a deal as it would only save you a step really but regardless, might be good to either remove that name or keep him there but change to a different solution.
</details>

**Comment by icguy** [comment]
Apparently you cannot keep these criminals from going town to town murdering everyone in sight.

**Comment by Joonpark13** [comment]
You're right - I made the suspect the same as kind of an homage to the game that inspired us in the first place, but I didn't think of how if you'd already played the CL mystery it might spoil things for you. I wouldn't mind if someone wanted to change the name of the suspect!
